darwin/torch/utils.py

```diff
@@ -11,6 +11,8 @@
 from darwin.dataset.identifier import DatasetIdentifier
 from darwin.datatypes import Segment
 
+NumpyCastable = Union[np.ndarray, torch.Tensor, List[Union[int, float]]]
```

Comment:
Numpy has types included in newer versions that can be used instead of `np.ndarray` but don't do it if it breaks anything.

`numpy.typing` contains:
* `ArrayLike`
* `DTypeLike`
* `NBitBase`
* `NDArray`

Reply:
Might give these a try, I suspect they'll cause issues because we use List[Union[int, float]] which might not be covered